modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/Bracket.kt

```diff
@@ -145,5 +145,5 @@ interface Bracket<F, E> : MonadError<F, E> {
    *
    */
   fun <A> Kind<F, A>.guaranteeCase(finalizer: (ExitCase<E>) -> Kind<F, Unit>): Kind<F, A> =
-    bracketCase({ _, e -> finalizer(e) }, { this })
+    just<Unit>(Unit).bracketCase({ _, e -> finalizer(e) }, { this })
```

Comment:
I believe we have `IO.unit` for this.

Reply:
Actually I was using `unit()` before which has a funny resolution. Anyway we're on Bracket, so I believe we need to stick with the explicit just like it's done above, no?

Reply:
I believe we have `lazy()` for this :trollface:

Reply:
I cannot use it here unless you mean the kotlin one. Anyway is there a problem from using `just` here?

Reply:
Awww, it's above in the hierarchy 

https://github.com/arrow-kt/arrow/blob/master/modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/MonadDefer.kt#L28

just works, ye